nautobot/core/api/metadata.py

```diff
@@ -279,17 +300,19 @@ def determine_metadata(self, request, view):
 
         return metadata
 
-    def add_missing_field_to_view_config_layout(self, view_config_layout, exclude_fields):
+    def add_missing_field_to_view_config_layout(self, view_config_layout, exclude_fields, advanced_fields):
         """Add fields from view serializer fields that are missing from view_config_layout."""
         serializer_fields = self.view_serializer.fields.keys()
         view_config_fields = [
             field for config in view_config_layout for field_list in config.values() for field in field_list["fields"]
         ]
-        missing_fields = sorted(set(serializer_fields) - set(view_config_fields) - set(exclude_fields))
+        missing_fields = sorted(
+            set(serializer_fields) - set(view_config_fields) - set(exclude_fields) - set(advanced_fields)
+        )
         view_config_layout[0]["Other Fields"] = {"fields": missing_fields}
         return view_config_layout
 
-    def restructure_view_config(self, view_config):
+    def restructure_view_config(self, serializer, view_config):
         """
         Restructure the view config by removing specific fields ("composite_key", "url", "display", "status", "id")
```

Comment:
You might want to include the new fields you added `created, updated` or remove the list entirely?